lib/components/form/advanced-settings-panel.tsx

```diff
@@ -135,15 +158,13 @@ const AdvancedSettingsPanel = ({
       <Subheader>
```

Comment:
I left them invisible for screenreaders since I think they're helpful context, and so some configurations could have access to them! I took them out, let me know what you think? I kinda miss them but I see what you mean.